app/src/components/steps/select-city-steps.tsx

```diff
@@ -197,6 +224,8 @@ export default function SelectCityStep({
     skip: cityInputQuery?.length <= 2,
   });
 
+  // somehow fetch the city,
```

Comment:
This would be good to do. Lmk if you have questions about the OpenClimate API here.